vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,214 @@
+#Version 0.3
+
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gmps: <http://linked.data.gov.au/def/qld-resource-permit-status/> .
+
+<http://linked.data.gov.au/def/qld-resource-permit-status> a owl:Ontology .
+
+gmps:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-09-13"^^xsd:date ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:source "https://myminesonline.business.qld.gov.au" ;
+    skos:definition "The stage in the life cycle of a Queensland Resource Permit."@en ;
+    skos:hasTopConcept gmps:application,
+        gmps:discarded,
+        gmps:granted,
+        gmps:non-current,
+        gmps:not-accepted,
+        gmps:prelodgement,
+        gmps:submitted ;
+    skos:prefLabel "Queensland Resource Permit Status"@en .
+
+gmps:abandoned a skos:Concept ;
+    skos:definition "The Resource Permit was not granted and has been abandoned by the applicant."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Abandoned"@en .
+
+gmps:accepted a skos:Concept ;
+    skos:definition "The application has been accepted."@en ;
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Accepted"@en .
+
+gmps:awaiting-payment-confirmation a skos:Concept ;
+    skos:definition "Payment has been made, but the Department needs confirmation of this."@en ;
+    skos:broader gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Awaiting payment confirmation"@en .
+
+gmps:competing-application a skos:Concept ;
+    skos:definition "This application represents a competing application."@en ;
+    skos:broader gmps:application,
+        gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Competing application"@en .
+
+gmps:conditionally-surrendered a skos:Concept ;
+    skos:definition "The application has been conditionally surrendered."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conditionally surrendered"@en .
+
+gmps:conversion-of-ml-to-mc a skos:Concept ;
```

Comment:
Something seems amiss with this. How common is it to downgrade from a Mining Lease to a Mining Claim? Given there are far greater restrictions on a Mining Claim. If we will have an audit trail of changes to and between permits why do we need this status?

Reply:
@johnmckellar did Jodie provide any answer to this?
@DavidCrosswellGSQ Will our system tracks changes, or record the 'ancestor' permit (as a provenance item) when permits are converted? If so this seems superfluous.

Reply:
Jodie said yesterday that under legislation, there was a period when this was permitted. There were a significant number of such conversions. The downgrade is no longer permitted. Essentially, it is an historical thing, but Jodie said that she wanted the term kept!!??

Reply:
As i haven't been in any of the meetings with Jodie can you please document any of the reasons given that address my questions and comments throughout this review? If there are valid reasons i'm happy to keep the terms, but these reasons need to be documented for posterity.